pages/about.html

```diff
@@ -9,34 +9,38 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
-    <title>Intro to Blockchain</title>
+    <title>Blockchain courses</title>
 </head>
 <body>
```

Comment:
The whole of the about page is breaking. Kindly fix this issue.
![image](https://user-images.githubusercontent.com/33268927/192084058-cd8814c7-457f-4f52-94ec-fbb86fa2c226.png)
![image](https://user-images.githubusercontent.com/33268927/192084012-b4241ee1-452f-4ca1-ba92-afaf5ab7738c.png)